plugins/kinet/KiNetNode.cpp

```diff
@@ -125,23 +134,33 @@ bool KiNetNode::SendDMX(const IPV4Address &target_ip, const DmxBuffer &buffer) {
 bool KiNetNode::SendPortOut(const IPV4Address &target_ip,
                             const uint8_t port,
                             const DmxBuffer &buffer) {
-  static const uint8_t flags = 0;
-  static const uint8_t padding = 0;
+  static const uint8_t flags1 = 0;  // Definitely flags of some sort
+  static const uint16_t flags2 = 0;  // Possibly always 0
   static const uint32_t universe = 0xffffffff;
 
   if (!buffer.Size()) {
     OLA_DEBUG << "Not sending 0 length packet";
     return true;
   }
 
+  uint16_t buffSize = static_cast<uint16_t>(buffer.Size());
+  uint16_t buffSizeRegulated = std::max(
+    buffSize,
+    KINET_PORTOUT_MIN_BUFFER_SIZE);
+
   m_output_queue.Clear();
   PopulatePacketHeader(KINET_PORTOUT_MSG);
-  m_output_stream << universe << port << padding
-                  << flags << padding  // Are the flags actually 16 bit?
-                  << static_cast<uint16_t>(buffer.Size());
+  m_output_stream << universe << port
+                  << flags1 << flags2
+                  << HostToNetwork(buffSizeRegulated);
```

Comment:
Matching code style:
```suggestion
                  << HostToNetwork(buffer_size_regulated);
```

Reply:
Fixed.